src/site/content/en/blog/sms-otp-form/index.md

```diff
@@ -156,8 +156,8 @@ Enhance the user experience of entering an OTP by aligning with
 [the origin-bound one-time codes delivered via SMS](https://wicg.github.io/sms-one-time-codes/) 
 specification. 
 
-The format rule is simple: Finish the
-SMS message with the domain preceded with `@` and the OTP preceded with `#`.
+The format rule is simple: Finish the SMS message with the receiver domain
```

Comment:
I'm not sure that this addition makes things clearer. Did you update this based on some feedback (and what was it)?

Reply:
Regarding this change, it's meant to clarify that the domain should be bound to the receiver domain rather than the sender domain in case they are different.